Annotation/Parameter.php

```diff
@@ -6,28 +6,25 @@
  * @Annotation
  * @Target({"PROPERTY", "CLASS"})
  */
-final class Parameter
+final class Parameter implements ParameterInterface
 {
     /**
      * NOTE: Trying to convert array to any type is invalid
-     * You can convert primitive types and string
+     * You can convert the data into primitive types and string
```

Comment:
`You can convert the data into primitive types` string'de bir primitive type.

Reply:
Java :)